From 7bc9b115baba6df2969451cd6907e2d7543ef22d Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Mon, 5 Aug 2024 10:18:31 +0100
Subject: [PATCH] Update common submodule

Richard W.M. Jones (1):
      mlcustomize: Write qemu-ga log file name to log.txt
---
 common | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Submodule common 04116678..7b84a370:
diff --git a/common/mlcustomize/inject_virtio_win.ml b/common/mlcustomize/inject_virtio_win.ml
index 2981bff5..afec1e45 100644
--- a/common/mlcustomize/inject_virtio_win.ml
+++ b/common/mlcustomize/inject_virtio_win.ml
@@ -606,6 +606,8 @@ and configure_qemu_ga t tempdir_win files =
   add "# Run qemu-ga installers";
   List.iter (
     fun msi ->
+      add (sprintf "Write-Host \"Writing log to %s\\%s.log\""
+             tempdir_win msi);
       (* [`] is an escape char for quotes *)
       add (sprintf "Start-Process -Wait -FilePath \"%s\\%s\" -ArgumentList \"/norestart\",\"/qn\",\"/l+*vx\",\"`\"%s\\%s.log`\"\""
              tempdir_win msi tempdir_win msi)
